sycl/test/basic_tests/parallel_for_range_host.cpp

```diff
@@ -0,0 +1,90 @@
+// RUN: %clangxx -fsycl %s -o %t.out
+// RUN: env SYCL_DEVICE_TYPE=HOST %t.out
+
```

Comment:
No LLVM project header

Reply:
No need to add copyrights for new tests if they are created by author - only if they are copied from non-LLVM project.